src/main/java/sg/christopher/duke/Duke.java

```diff
@@ -0,0 +1,273 @@
+package sg.christopher.duke;
+
+import javafx.application.Platform;
+import javafx.stage.Stage;
+import sg.christopher.duke.entities.Deadline;
+import sg.christopher.duke.entities.Event;
+import sg.christopher.duke.entities.Task;
+import sg.christopher.duke.entities.Todo;
+import sg.christopher.duke.io.DataManager;
+import sg.christopher.duke.ui.MainWindow;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Duke {
+
+    private static List<Task> savedItems = loadSavedItems();
+
+    private MainWindow mainWindowController;
```

Comment:
not related to Code Standards, but may be good to break dependency of Duke on MainWindow though some level of abstraction.